en/SettingsView/General.md

```diff
@@ -27,51 +27,71 @@ This section defines a number of miscellaneous settings, related to (non exhaust
 ![Miscellaneous settings](../../assets/settings/general/miscellaneous.jpg)
 
 The settings are:
+- <span id="language"></span>**Language**: System (System Language) | Bulgarian, Chinese, ...
+
+  ![Display languages](../../assets/settings/general/languages.jpg)
 - <span id="colour_scheme"></span>**Color Scheme**: Indoor (Dark) | Outdoor (Light)
-- **Map Provider**: Bing | Google | Statkart | Eniro
-- **Map Type**: Hybrid | Street | Satellite
-- **Stream GCS Position**: Never | Always | When in Follow Me flight mode.
-- **Font Size**: Font size across app (Requires restart)
-- **Mute all audio output**: Turns off all audio output. 
-- <span id="autosave_log"></span>**Save telemetry log after each flight**: Logs automatically saved to the *Application Load/Save Path* ([below](#load_save_path)) after flight. 
-- **Save telemetry log even if vehicle was not armed**: Logs when a vehicle connects to *QGroundControl*. 
-  Stops logging when the last vehicle disconnects.
-- **Use preflight checklist**: Enable pre-flight checklist.
-- **Virtual Joystick**: Enable [virtual joysticks](../SettingsView/VirtualJoystick.md) (PX4 only)
+- **Map Provider**: Google | Mapbox | Bing | Airmap | VWorld | Eniro | Statkart
+- **Map Type**: Road | Hybrid | Satellite
+- **Stream GCS Position**: Never | Always | When in Follow Me Flight Mode.
+- **UI Scaling**: UI scale percentage (affects fonts, icons, button sizes, layout etc.) 
+- **Mute all audio output**: Turns off all audio output.
+- **Check for Internet Connection**: TBD
```

Comment:
@DonLakeFlyer What is check for internet connection for/how does it work?

Reply:
That would be @dogmaphobic. First I'm seeing that one.

Reply:
QGC needs Internet for downloading map tiles. If you are not connected to the Internet, the map tile engine will keep trying to download and failing. To stop it from doing that, we check to see if you actually have internet. If you don't the map tile engine is disabled. 

The problem is that the way we test for Internet doesn't work in China. It always returns negative (telling you don't have internet) and therefore, maps are disabled. This was a way to allow Chinese to use maps. If this setting is set, we skip the tests and assume Internet is present.

Reply:
Change the name to something which talks about map download as well?

Reply:
Maybe "Map Download Internet Check"?

I have changed to 

> - **Check for Internet Connection**: Uncheck to allow maps to be used in China/places where map tile downloads are likely to fail (stops the map-tile engine continually rechecking for an Internet connection).